### PR TITLE
Dependabot

### PR DESCRIPTION
**Pull request by NlockTime**

### CODE REVIEW

**Comment by NlockTime** [comment]
+

**Comment by NlockTime** [comment]
pipeline and dependabot

**Review by NlockTime — COMMENTED** [review]
pipeline and dependabot

**Review by NlockTime — COMMENTED** [review]
Done!

File: .codesandbox/tasks.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (utxo-detective) [inline .codesandbox/tasks.json:1]: are `.codesandbox/tasks.json` and `.devcontainer` needed for dependabot?

**Review by utxo-detective — CHANGES_REQUESTED** [review]

File: .codesandbox/tasks.json (hunk 1 above)
Comment (NlockTime) [reply .codesandbox/tasks.json:1]: No.

Comment (utxo-detective) [reply .codesandbox/tasks.json:1]: Can you remove the extra files?

**Pull request closed without merging**